Gemfile

```diff
@@ -4,15 +4,18 @@ source 'https://rubygems.org'
 gem 'berkshelf', '~> 3.0'
 
 # Install omnibus software
-gem 'omnibus', :github => 'chef/omnibus', :branch => '7.0.9'
-gem 'omnibus-software', :github => 'chef/omnibus-software' #, :branch => 'omnibus/3.2-stable'
+gem 'omnibus', :github => 'chef/omnibus'
+gem 'omnibus-software', :github => 'chef/omnibus-software'
+
+gem 'ohai', '>17.7'
```

Comment:
Due to:

```
Error:

    ERROR:  Error installing /main/google-fluentd/plugin_gems/21-fluent-plugin-kubernetes_metadata_filter-2.5.2.gem:
	The last version of kubeclient (< 5) to support your Ruby & RubyGems was 4.10.1. Try installing it with `gem install kubeclient -v 4.10.1` and then running the current command again
	kubeclient requires Ruby version >= 2.7.0. The current ruby version is 2.6.10.210.
```

We shall also add `gem 'kubeclient', '4.10.1'` (I tested locally in a docker image, it worked)